src/renderer/services/dex.js

```diff
@@ -29,6 +29,17 @@ const TX_ATTR_USAGE_WITHDRAW_VALIDUNTIL = 0xA6;
 const SIGNATUREREQUESTTYPE_WITHDRAWSTEP_MARK = '91';
 const SIGNATUREREQUESTTYPE_WITHDRAWSTEP_WITHDRAW = '92';
 const SIGNATUREREQUESTTYPE_CLAIM_GAS = '94';
+const binSizeToBinCountMap = {
+  1: 1440, //  1 Min ~ 1 day
+  5: 1440, //  5 Min ~ 5 days
+  15: 1440, // 15 Min ~ 15 days
+  30: 1440, // 30 Min ~ 30 days
+  60: 1440, // 1 Hrs ~ 60 days
+  360: 1440, // 6 Hrs ~ 360 days
+  1440: 1095, // 1 Day ~ 3 yrs
+  4320: 1825, // 3 Days ~ 5 yrs
+  10080: 1825, // 1 Week ~ 5 yrs
```

Comment:
If this gets 1825 buckets and the previous one also get's 1825 buckets; how are they both 5 years worth?

Reply:
Fixing. Should = 10080: 520